api/api.proto

```diff
@@ -2163,24 +2171,32 @@ message SdkCloudBackupStatus {
   google.protobuf.Timestamp completed_time = 6;
   // NodeID is the ID of the node where this Op is active
   string node_id = 7;
+  // SourceVolumeID is the the volume that is either being backed up to cloud
+  // or target volume to which cloudsnap is being restored
+  string src_volume_id = 8;
+  // ErrString indicates the failure cause for failed backup/restore
+  string err_string = 9;
 }
 
 // Defines a request to retreive the status of a backup or restore for a
 // specified volume
 message SdkCloudBackupStatusRequest {
   // This is an optional value which is used to get information on the
-  // status of a backup for the specified volume. If no volume id is provided,
-  // then status for all volumes is returned.
+  // status of a backup for the specified volume. If no volume id and TaskId
+  // is provided, then status for all volumes is returned.
   string volume_id = 1;
   // Local indicates if only those backups/restores that are
   // active on current node must be returned
   bool local = 2;
+  // TaskId is the backup/restore task id, if this is specified,
+  // VolumeId is ignored.
```

Comment:
this should be `volume_id`

Reply:
other fields are not that way..

Reply:
In Golang it is called VolumeId but in other languages and in the proto file it is `volume_id`. In the documentation generated with this proto file, we should use proto names as links.